src/ft2font.cpp

```diff
@@ -560,14 +566,12 @@ void FT2Font::clear()
 
 void FT2Font::set_size(double ptsize, double dpi)
 {
-    int error = FT_Set_Char_Size(
-        face, (long)(ptsize * 64), 0, (unsigned int)(dpi * hinting_factor), (unsigned int)dpi);
+    if (int error = FT_Set_Char_Size(
+        face, (long)(ptsize * 64), 0, (unsigned int)(dpi * hinting_factor), (unsigned int)dpi)) {
```

Comment:
Not sure I like this inlined call with the alignment here.

Reply:
sure, moved it out